rest/src/main/java/discord4j/rest/entity/RestGuild.java

```diff
@@ -64,11 +64,14 @@ static RestGuild create(RestClient restClient, long id) {
     /**
      * Retrieve this guild's data upon subscription.
      *
+     * @param withCounts when true, will return approximate member and presence counts for the guild
      * @return a {@link Mono} where, upon successful completion, emits the {@link GuildUpdateData} belonging to this
      * entity. If an error is received, it is emitted through the {@code Mono}.
      */
-    public Mono<GuildUpdateData> getData() {
-        return restClient.getGuildService().getGuild(id);
+    public Mono<GuildUpdateData> getData(@Nullable Boolean withCounts) {
```

Comment:
Keep `getData()` and make it call `getData(true)`

Reply:
I think the default value is false according to the doc

Reply:
Yeah, I think @Shadorc is right.

Reply:
Alright, let's make it `getData(false)` to maintain 1:1 behavior against API

Reply:
Actually the call should just hit the plain endpoint, so.. null?

Reply:
> ```
> git rebase --onto 3.1.x master feature/with_counts
> ```
This does not work for me. Output: `Does not point to a valid commit '3.1.x'` 😕

Reply:
Hm, you might need to checkout 3.1.x locally. Forks only contain master I believe

Reply:
> Hm, you might need to checkout 3.1.x locally. Forks only contain master I believe

Just worked! Thank you. 😉

Reply:
> Alright, let's make it `getData(false)` to maintain 1:1 behavior against API

So the query parameter is fix? Or could the user input true too?

Reply:
This would only apply to the existing `getData()` method implementation, so that we don't break that method in 3.1.x branch.

The end result to have `getData()` call `getData(null)`, but keeping the `getData(Boolean)` implementation as you have now.

Reply:
`public Mono<GuildUpdateData> getData()` must remain alongside `public Mono<GuildUpdateData> getData(@Nullable Boolean withCounts)`. Don't delete either of them, add getData so we can keep both.

The former should have this implementation

```java
public Mono<GuildUpdateData> getData() {
    return getData(null);
}
```

Please include javadoc to both and make sure to add the parameter javadoc to the withCounts one.